docs/src/main/tut/datatypes/NonEmptyList.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: docs
+title:  "NonEmptyList"
+section: "data"
+source: "core/src/main/scala/cats/data/NonEmptyList.scala"
+scaladoc: "#cats.data.NonEmptyList"
+---
+# NonEmptyList
+
+If you have had the opportunity of taking a look to [Validated](validated.html) or [IOR](ior.html), you'll find that one of the most common case is to use `NonEmptyList` with one of these data structures.
```

Comment:
s/one of the most/a/

Reply:
Thank you. I'll fix all the typos that you've found.